### PR TITLE
Update smartmontools to 6.4

### PR DESCRIPTION
**Pull request by samm-git**
Signed-off-by: Alex Samorukov <samm@os2.kiev.ua>

### CODE REVIEW

File: Library/Formula/smartmontools.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@
 class Smartmontools < Formula
   desc "SMART hard drive monitoring"
   homepage 'http://sourceforge.net/apps/trac/smartmontools/'
```
Comment (vszakats) [inline Library/Formula/smartmontools.rb]: This URL is no longer valid; would you update it to its permanent redirect (+https) ?:
`homepage "https://sourceforge.net/projects/smartmontools/"`

Comment (samm-git) [inline Library/Formula/smartmontools.rb]: Correct homepage URL is http://smartmontools.org/. Sourceforge is used only as SVN server and file storage.

Comment (vszakats) [inline Library/Formula/smartmontools.rb]: Even better! I'd suggest using the https version of that as well:
`homepage "https://www.smartmontools.org"`

Comment (samm-git) [inline Library/Formula/smartmontools.rb]: I am ok with https. Should i update pull request?

Comment (vszakats) [inline Library/Formula/smartmontools.rb]: Yes, please!

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/3acb81804ae0d9de04b48b6a949099e7ec41f351. Thanks for your contribution to Homebrew, we appreciate it! :smiley_cat:.

Please note for future the preferred commit message for version bumps is `smartmontools 6.4`, Thanks!

**Pull request closed without merging**